spec/controllers/api/v1/orders_controller_spec.rb

```diff
@@ -251,6 +252,14 @@ def create_order_with_transport(state, opts = {})
           end
         end
 
+        it "does not return valid order type when type is other" do
+          create :online_order, :awaiting_dispatch, description: "Testing", order_transport: ggv_transport
+          get :index, params: { type: "other" }
+          expect(response.status).to eq(200)
+          expect(parsed_body["designations"].count).to eq(0)
```

Comment:
instead of count / total pages,
it would be good to test if all the response object has detail_type only as 'other'

````
order_types = response['designations'].map { |res| res['detail_type'] }
expect(order_types.uniq).to match_array(['other'])
```